lib/concordance-options.js

```diff
@@ -85,17 +85,21 @@ const colorTheme = {
 	undefined: ansiStyles.yellow,
 };
 
-const plainTheme = JSON.parse(JSON.stringify(colorTheme), value => typeof value === 'string' ? stripAnsi(value) : value);
-
-const theme = chalk.level > 0 ? colorTheme : plainTheme;
+const plainTheme = JSON.parse(JSON.stringify(colorTheme), (_name, value) => typeof value === 'string' ? stripAnsi(value) : value);
```

Comment:
I'm confused, how did `plainTheme` _ever_ work?!

Reply:
Pretty much on accident. It ended up as an empty string, which concordance is sufficiently tolerant to accept as effectively absent, and the ava `colorTheme` stripped of its ANSI escape sequences doesn't actually deviate from the concordance default theme:
<details><summary>diff</summary>

```diff
--- concordance defaultTheme (normalized)
+++ ava corrected plainTheme (normalized)
@@ -1,11 +1,7 @@
 {
-  "bigInt": {
-    "close": "",
-    "open": ""
-  },
   "boolean": {
     "close": "",
     "open": ""
   },
   "circular": "[Circular]",
   "date": {
@@ -42,23 +38,20 @@
   },
   "global": {
     "close": "",
     "open": ""
   },
   "item": {
-    "after": ",",
-    "customFormat": null,
-    "increaseValueIndent": false
+    "after": ","
   "list": {
     "closeBracket": "]",
     "openBracket": "["
   },
   "mapEntry": {
-    "after": ",",
-    "separator": " => "
+    "after": ","
   },
   "maxDepth": "…",
   "null": {
     "close": "",
     "open": ""
   },
@@ -81,26 +74,23 @@
       "close": "",
       "open": "@"
     }
   },
   "property": {
     "after": ",",
-    "customFormat": null,
-    "increaseValueIndent": false,
     "keyBracket": {
       "close": "]",
       "open": "["
     },
-    "separator": ": "
+    "valueFallback": "…"
   },
   "regexp": {
     "flags": {
       "close": "",
       "open": ""
     },
-    "separator": "---",
     "source": {
       "close": "/",
       "open": "/"
     }
   },
   "stats": {
@@ -133,18 +123,16 @@
         "close": "",
         "open": ""
       }
     },
     "line": {
       "close": "'",
-      "escapeQuote": "'",
       "open": "'"
     },
     "multiline": {
       "end": "`",
-      "escapeQuote": "`",
       "start": "`"
     },
     "open": ""
   },
   "symbol": {
     "close": "",
```

</details>